app/code/community/Algolia/Algoliasearch/Model/Queue.php

```diff
@@ -416,4 +463,12 @@ private function clearOldLogRecords()
             $this->db->query("DELETE FROM {$this->logTable} WHERE id IN (" . implode(", ", $idsToDelete) . ")");
         }
     }
+
+    public function clearQueue($canClear = false)
```

Comment:
Why the method has the argument `$canClear`?

Reply:
@JanPetr It seems unnecessary because it might be. I added it there for intent because I'm not comfortable with a method that is free to just truncate a table. Outside of that, there are benefits to adding the parameter incase there is logic used to pass into this method. I'm happy to remove it though if you think it's unnecessary!